index.md

```diff
@@ -1,4 +1,127 @@
-# This is the **HOMEPAGE**.
-Refer to [Markdown](http://daringfireball.net/projects/markdown/) for how to write markdown files.
-## Quick Start Notes:
-1. Add images to the *images* folder if the file is referencing an image.
+---
+title: Welcome to the Blazor preview docs website!
+description: Learn about Blazor, an experimental .NET web framework using C#/Razor and HTML that runs in the browser with WebAssembly.
+---
+<div class="cr cr-top cr-right cr-sticky cr-green">Experimental</div>
+<div class="hero">
+  <div class="wrap">
+    <div class="text">
+      <strong>BLAZOR</strong>
+    </div>
+    <div class="buttons-unit-small">
+      <a class="version-link" href="#">Get Started</a><span>|</span><a class="github-link" href="/docs/get-started.html">View in Github</a>
+    </div>
+    <div class="minitext">
+        Blazor is an experimental .NET web framework using C#/Razor and HTML that runs in the browser with WebAssembly.
+    </div>
+    <div class="buttons-unit">
+      <a href="/docs/get-started.html" class="button"><i class="glyphicon glyphicon-send"></i>Get Started</a>
+    </div>
+  </div>
+</div>
+<div class="key-section">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <i class="glyphicon glyphicon-wrench"></i>
+        <section>
+          <h2>Build a Web UI with C#</h2>
+          <p class="lead">Blazor is a web UI framework, similar to existing JavaScript frameworks such as Angular and React, but you write C# instead of JavaScript.</p>
+          <a href="https://github.com/aspnet/Blazor/wiki/FAQ" class="btn btn-primary">What is Blazor?</a>
+        </section>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="counter-key-section">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <section>
+          <h2>Full Stack .NET</h2>
+          <p class="lead">Do full-stack .NET development using stable and consistent tools, languages and APIs both in the browser and on the server.</p>
+          <a href="https://www.microsoft.com/net" class="btn btn-primary">Learn more about the .NET platform</a>
+        </section>
+        <i class="glyphicon glyphicon-tasks"></i>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="key-section">
+  <div class="container content">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <i class="glyphicon glyphicon-globe"></i>
+        <section>
+          <h2>Runs in all browsers and implements</h2>
```

Comment:
This is the one where I was asking about adding ".NET Standard" to the end of this heading.